lib/controllers/base.js

```diff
@@ -55,18 +56,21 @@ Controller.prototype.route = function() {
 
   // NOTE: is there a better place to do this mapping?
   var pathOptions = endpoint.string;
-  if (app.name === 'restify') {
-    if (self.method === 'delete')
+  if (this.isRestify) {
+    if (self.method === 'delete') {
```

Comment:
How do you think if I check by using this command `typeof app[self.method] === 'function'`, restify server does not have `delete` method, and express does not have 'del', we can use them